Content/Items/Consumables/Maps/CaveMap.cs

```diff
@@ -2,14 +2,16 @@
 
 namespace PathOfTerraria.Content.Items.Consumables.Maps;
 
-public class CaveMap : Map
+internal class CaveMap : Map
 {
 	public int RoomSizeMin = 14;
 	public int RoomSizeMax = 22;
 	public int BossRoomSize = 30;
 	public int SpawnRoomSize = 10;
 	public int ExtraRoomDist = 10;
 
+	public override float DropChance => 1f;
```

Comment:
We may need to add a ItemLevelReq, nbd now but to remember in the future

Reply:
MinDropItemLevel in PoTItem